app/main.py

```diff
@@ -1,8 +1,32 @@
+from __future__ import annotations
+
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(data["name"], data["age"]) for data in people]
```

Comment:
Create a list here in which, during the first iteration, you add Person objects without the spouse field, and in the second iteration, add them.